### ISSUE TITLE
Write CWL and Docker for Picard SamToFastq

### ISSUE THREAD

**Issue by rhshah** [msk-access/cwl-commandlinetools#48]
Tool: https://gatk.broadinstitute.org/hc/en-us/articles/360047218131-SamToFastq-Picard-
Version: GATK4 4.1.8.1

Example Command:
```bash
/opt/common/CentOS_7/java/jdk1.8.0_202/bin/java -Djava.io.tmpdir=/scratch/ -Xmx4g -jar /opt/common/CentOS_7/picard/picard-2.9.0-jar/picard.jar SamToFastq I=/dev/stdin F=/juno/work/bergerm1/bergerlab/arorak/Project_benchmark_fgbio/FqToBamFgbio_2020-06-10/NovaSeq_Plasma_Healthy_Donor/DONOR11-N/DONOR11-N.fqtobam.clip.fastq.gz INTERLEAVE=true CLIPPING_ATTRIBUTE=XT CLIPPING_ACTION=X CLIPPING_MIN_LENGTH=25
```

- [x] CWL
- [x] Docker
- [x] README
- [x] Example YAML
- [ ] Test with cwltool and toil

**Comment by murphycj2** [comment]
i'll take this

**Comment by rhshah** [comment]
#69 Resolves #48
